static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -252,10 +253,16 @@ export class SentryAppExternalForm extends Component<Props, State> {
 
     if (fieldToPass.type === 'select') {
       // find the options from state to pass down
-      const defaultOptions = (field.choices || []).map(([value, label]) => ({
+      let defaultOptions = (field.choices || []).map(([value, label]) => ({
         value,
         label,
       }));
+      // there are no choices if skip_load_on_open is set, just use what's in the defaultResetValue
+      if (field.skip_load_on_open === true) {
+        defaultOptions = [
+          {value: defaultResetValues[field.name], label: defaultResetValues[field.name]},
```

Comment:
Just double checking here, we're using the same value for both value and label. Will this cause issues if they don't match? i.e. API returns {value: "23" , label: "Project #23"}, when we reopen the form, will it still say "Project #23", or just the value that gets sent to the API?

Reply:
Oof that's a good point. We only store the value so there's no way to get the label the API would return if it's different 😬